CONTRIBUTING.md

```diff
@@ -488,7 +501,11 @@ Unlike the user setting overrides, not all of these environment variables have t
 -   `__AMAZONQLSP_MANIFEST_URL`: for aws.dev.amazonqLsp.manifestUrl
 -   `__AMAZONQLSP_SUPPORTED_VERSIONS`: for aws.dev.amazonqLsp.supportedVersions
 -   `__AMAZONQLSP_ID`: for aws.dev.amazonqLsp.id
--   `__AMAZONQLSP_LOCATION_OVERRIDE`: for aws.dev.amazonqLsp.locationOverride
+-   `__AMAZONQLSP_LOCATION_OVERRIDE`: for aws.dev.amazonqWorkspaceLsp.locationOverride
+-   `__AMAZONQWORKSPACELSP_MANIFEST_URL`: for aws.dev.amazonqWorkspaceLsp.manifestUrl
+-   `__AMAZONQWORKSPACELSP_SUPPORTED_VERSIONS`: for aws.dev.amazonqWorkspaceLsp.supportedVersions
+-   `__AMAZONQWORKSPACELSP_ID`: for aws.dev.amazonqWorkspaceLsp.id
+-   `__AMAZONQWORKSPACELSP_LOCATION_OVERRIDE`: for aws.dev.amazonqWorkspaceLsp.locationOverride
```

Comment:
`_OVERRIDE` can be dropped in these names. All env vars are "overrides" :)

People tend to copy existing stuff, and may start adding _OVERRIDE to all the env vars.